docs/application/native/guides/security/privacy-related-permissions.md

```diff
@@ -128,7 +128,105 @@ To check whether an application has permission to use a privilege, and to reques
+
+     ```
+                case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_DENY:
+                    /* Show a message and terminate the application */
+                    break;
+
+     ```
+
+   - If the result value is `PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ASK`, the application must request permission from the user with the `ppm_request_permissions()` function, which displays a dialog box. You can pass user data as the forth parameter of the function, as a pointer to an application context instance. When the user makes a decision, a callback defined as the second parameter is invoked.
+
+     The dialog box asking for user permissions is shown only if the `ppm_request_permissions()` function returns without an error.
+
+    ```
+                case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ASK:
+                   askable_privileges[askable_privileges_count++] = privileges[it];
+                   /* Log and handle errors */
+                   break;
+                }
+            }
+            ret = ppm_request_permissions(askable_privileges, askable_privileges_count,
+                                          app_request_multiple_response_cb, NULL);
+        } else {
+            /* ret != PRIVACY_PRIVILEGE_MANAGER_ERROR_NONE */
+            /* Handle errors */
+        }
+    }
+    ```
+
+3. If you need to request user permissions, handle the user decisions within the callback.
+
+   The user decisions is returned in the callback as a value of the `ppm_request_result_e` enumeration (in [mobile](../../api/mobile/latest/group__CAPI__PRIVACY__PRIVILEGE__MANAGER__MODULE.html#ga0be419f8be0a398fd6e9af29f9c3e29b) and [wearable](../../api/wearable/latest/group__CAPI__PRIVACY__PRIVILEGE__MANAGER__MODULE.html#ga0be419f8be0a398fd6e9af29f9c3e29b) applications).
+
+    ```
+    void
+    app_request_multiple_response_cb(ppm_call_cause_e cause, ppm_request_result_e* results,
+                                     const char **privileges, size_t privileges_count, void *user_data)
+    {
+        if (cause == PRIVACY_PRIVILEGE_MANAGER_CALL_CAUSE_ERROR) {
+            /* Log and handle errors */
+
+            return;
+        }
+        for (int it = 0; it < privileges_count; ++it) {
+            switch (results[it]) {
+                case PRIVACY_PRIVILEGE_MANAGER_REQUEST_RESULT_ALLOW_FOREVER:
+                    /* Update UI and start accessing protected functionality */
+                    break;
+                case PRIVACY_PRIVILEGE_MANAGER_REQUEST_RESULT_DENY_FOREVER:
+                    /* Show a message and terminate the application */
+                    break;
+                case PRIVACY_PRIVILEGE_MANAGER_REQUEST_RESULT_DENY_ONCE:
+                    /* Show a message with explanation */
+                    break;
+            }
+        }
+    }
+    ```
+
+## Note
```

Comment:
Please use the following style for note:

>**Note**
>
>Multiple privileges in single API call are supported from Tizen 5.0.

Reply:
Done